priv/templates/job_detail.html.eex

```diff
@@ -36,10 +36,11 @@ Raw Run Times shows all individual recorded run times in the order they were rec
     <script>
       var measurements = <%= job_json %>;
       var runTimes     = measurements.run_times;
+      var statistics   = measurements.statistics;
 
       var inputHeadline = "<%= input_headline(input_name) %>"
 
-      drawRawRunTimeCharts(runTimes, inputHeadline);
+      drawRawRunTimeCharts(runTimes, inputHeadline, statistics);
```

Comment:
It looks like in your function definition you put the `statistics` as the second positional argument. I think having it as the third argument makes sense, but you'll need to update your function definition [here](https://github.com/PragTob/benchee_html/pull/32/files#diff-885568f02ebb09eb4c00bea2cfc06989R89).

Reply:
Yeah I went back and forth on which to put first but I believe that the `job_detail.html.eex` reads better with the parameters ordered this way.